internal/rbd/replicationcontrollerserver.go

```diff
@@ -487,6 +488,29 @@ func (rs *ReplicationServer) DemoteVolume(ctx context.Context,
 	return &replication.DemoteVolumeResponse{}, nil
 }
 
+// This function checks the state of the remote cluster.
+// It returns true if the statue is up and unknown.
+func checkSiteStatus(ctx context.Context, mirrorStatus *librbd.GlobalMirrorImageStatus) bool {
```

Comment:
function comment should start with the function name
```
checkSiteStatus checks the state ...
```

Reply:
the function name can be renamed to checkRemoteSiteStatus?

Reply:
Agreeing with this, these are the only nits that I would point out as well.

Reply:
done.